.htaccess

```diff
@@ -17,6 +17,15 @@ RedirectMatch permanent "/lint.html" "/ecosystem.html"
 
 RedirectMatch permanent "/automation.html" "/platform.html"
 
+# Redirect the controller-automation pages to their respective index pages
+
+RedirectMatch permanent "automation-controller/(4.5|latest)/html/administration/(.*)$" "automation-controller/latest/html/administration/index.html"
+RedirectMatch permanent "automation-controller/(4.5|latest)/html/controllerapi/(.*)$" "automation-controller/latest/html/controllerapi/index.html"
+RedirectMatch permanent "automation-controller/(4.5|latest)/html/quickstart/(.*)$" "automation-controller/latest/html/quickstart/index.html"
+RedirectMatch permanent "automation-controller/(4.5|latest)/html/towerapi/(.*)$" "automation-controller/latest/html/controllerapi/index.html"
+RedirectMatch permanent "automation-controller/(4.5|latest)/html/upgrade-migration-guide/(.*)$" "automation-controller/latest/html/upgrade-migration-guide/index.html"
+RedirectMatch permanent "automation-controller/(4.5|latest)/html/userguide/(.*)$" "automation-controller/latest/html/userguide/index.html"
+
```

Comment:
This is where my redirect knowledge is a bit weak. I think this means for example, any userguide page will be redirected to the index page.... but what happens to the actual index page if it's constantly hitting this redirect? Will it just keep on redirecting in a forever loop?

Reply:
The index.html page shouldn't redirect to itself with this regex because the `$` symbol asserts the end of the url. The `(.*)` part catches any combination of characters before that end. Does that make sense? Not sure if I explain it well. Might be worth throwing up on the test server to verify?

Reply:
You know I was playing around with it in https://htaccess.madewithlove.com/ and I might be wrong. I guess we could update the regex to something like `(?!index\.html)(.*)$` to make sure `index.html` doesn't redirect to itself.

Reply:
I ran the rest build but I can't seem to get to the pages - I might have missed something... but it gives me an internal error... does anyone else get this?

Reply:
just to check but did you use the docsite build from your branch?

Reply:
@oraNod - yes I did
![image](https://github.com/ansible/docsite/assets/7514801/ce919a95-0f34-47ee-8990-53da2158ac93)

Reply:
@tvo318 On the test server (`docs.testing.ansible.com`) I see only three HTML directories for latest and 4.5. Trying to reach the other HTML directories like `administration` results in an HTTP 500 error.

![image](https://github.com/ansible/docsite/assets/32749632/593e7d82-7367-4c9f-82e9-423517680938)

![image](https://github.com/ansible/docsite/assets/32749632/073fcaaa-5f5b-42bd-aa78-adda95ae7b01)

Reply:
@oraNod Yes, that is the same with what I see. I wonder if we merge, on production we will be able to get the redirects or we'll see the same? Because the bookmarks are still showing old content. It is not clear where they are coming from.